app/src/main/java/com/androiddev/social/auth/data/UserManager.kt

```diff
@@ -10,7 +10,7 @@ import javax.inject.Inject
 
 interface UserManager {
     fun userComponentFor(accessTokenRequest: AccessTokenRequest): UserComponent
-    fun userComponentFor(code: String): UserComponent
+    fun userComponentFor(code: String): UserComponent?
```

Comment:
When would you want this to return null

Reply:
Sorry! I missed your comments!

It'll be null when it's not cached, for instance, after process death.

Reply:
You are right about the API of the UserManager, so I tried to fix it in this way [8cf8052](https://github.com/digitalbuddha/Firefly/pull/16/commits/8cf8052b660a4eb87d34c1bb33c1753894b20a0c). Let me know if this works.